frontend/components/site/siteBuildLogs.jsx

```diff
@@ -26,6 +52,11 @@ const SiteBuildLogs = () => {
           <li><DownloadBuildLogsButton buildId={buildId} buildLogsData={logs} /></li>
         </ul>
       </div>
+      {buildTooOld(buildDetails) && (
+      <div>
+        <SiteBuildLogTable buildLogs={['Builds more than 180 days old are deleted according to platform policy.']} />
+      </div>
+      )}
       {(!logs || logs?.length === 0) && (
```

Comment:
maybe something like this?
```suggestion
      {(!logs || logs?.length === 0) && (
        <div>
          {buildTooOld(buildDetails) && (
            <SiteBuildLogTable buildLogs={['Builds more than 180 days old are deleted according to platform policy.']} />
          )}
          {!buildTooOld(buildDetails) && (
            <SiteBuildLogTable buildLogs={['This build does not have any build logs.']} />
          )}
        </div>
      )}
```